pages/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/new/index.md

```diff
@@ -82,7 +82,7 @@ When you use existing infrastructure, DKP does _not_ create, modify, or delete t
     --subnet-ids=${AWS_SUBNET_IDS} \
     --internal-load-balancer=true \
     --additional-security-group-ids=${AWS_ADDITIONAL_SECURITY_GROUPS} \
-    --registry-mirror-url=${DOCKER_REGISTRY_ADDRESS} \
+    --registry-mirror-url=${DOCKER_REGISTRY_URL} \
```

Comment:
The JIRA is confusing!  This is ok,  but we need to fix the two other places on this page (and all the others, of course) that also mention DOCKER_REGISTRY_ADDRESS:   i.e. on line 65 and Line 71 above....

Reply:
Sorry I should have been clearer. We need to change `--registry-mirror-url=${DOCKER_REGISTRY_ADDRESS}` to `--registry-mirror-url=${DOCKER_REGISTRY_URL}` (like you've already done.
But whenever this variable `DOCKER_REGISTRY_ADDRESS` gets used in these pages you also need to change that to `DOCKER_REGISTRY_URL`.

What you don't want to change is other occurrences of `DOCKER_REGISTRY_ADDRESS` in other pages where its being used correctly, e.g. https://docs.d2iq.com/dkp/konvoy/2.2/choose-infrastructure/aws/air-gapped/seed-a-registry/

Reply:
Aha, so the example I sent you in slack earlier was one that does NOT get changed, but go ahead on the preprogram air-gapped create cluster and seed registry sections and change those.